Tests/Tests/UIColorExtensionsSpec.swift

```diff
@@ -1,3 +1,4 @@
+#if os(iOS) || os(watchOS)
```

Comment:
UIColor is available on WatchOS:
https://developer.apple.com/documentation/uikit/uicolor